### PR TITLE
Adding minor size optimizations

### PR DESCRIPTION
**Pull request by blockiosaurus**
Adding size reductions to token metadata in the form of abbreviated debug messages or removals where unnecessary.

### CODE REVIEW

**Review by samuelvanderwaal — APPROVED** [review]

**Review by samuelvanderwaal — APPROVED** [review]

File: token-metadata/program/src/processor/mod.rs [hunk 1]
```diff
@@ -201,39 +201,39 @@ fn process_legacy_instruction<'a>(
             )
         }
         MetadataInstruction::DeprecatedCreateMasterEdition(_args) => {
-            msg!("Instruction: Deprecated Create Master Edition, Removed in 1.1.0");
+            msg!("IX: Deprecated Create Master Edition, Removed in 1.1.0");
             Err(MetadataError::Removed.into())
         }
         MetadataInstruction::DeprecatedMintNewEditionFromMasterEditionViaPrintingToken => {
-            msg!("Instruction: Deprecated Mint New Edition from Master Edition Via Token, Removed in 1.1.0");
+            msg!("IX: Deprecated Mint New Edition from Master Edition Via Token, Removed in 1.1.0");
```
Comment (febo) [inline token-metadata/program/src/processor/mod.rs:208]: For the ones that have been removed, perhaps we can remove the whole message – not necessarily now.

Comment (blockiosaurus) [reply token-metadata/program/src/processor/mod.rs:208]: Yeah that's a good call. I didn't notice how verbose those messages are until today.

**Review by febo — APPROVED** [review]
This PR removes and shorten debug messages to reduce the binary size.

File: token-metadata/program/src/processor/mod.rs (hunk 1 above)
Comment (samuelvanderwaal) [reply token-metadata/program/src/processor/mod.rs:208]: Good point

**Pull request merged**